app/migrations/20240314122321-add_scope_and_subsector_to_userfile.cjs

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.addColumn('UserFile', 'scopes', {
+      type: Sequelize.STRING,
+      allowNull: true
+    })
+    await queryInterface.addColumn('UserFile', 'subsectors', {
```

Comment:
How are you storing multiple values in a single string? Same for scopes?
Please don't use comma separated values etc. in a database, that just makes it much harder to query for a specific one.